tests/test_column_files.py

```diff
@@ -31,7 +31,7 @@ def test_write_empty(self) -> None:
             s3_path_factory=path_factory,
         ):
             pass
```

Comment:
Below let's update the unit tests to use the non-S3 specific function names that have been added to the `AbstractDataStorage` class in this PR.